pkgs/desktops/cinnamon/cjs/default.nix

```
@@ -78,6 +78,7 @@ stdenv.mkDerivation rec {
       This module contains JavaScript bindings based on gobject-introspection.
     '';
 
+    license = [ licenses.gpl2 licenses.lgpl2 ];
```

the licensing here is pretty hairy
```suggestion
    license = with licenses; [
     gpl2Plus
     lgpl2Plus
     mit
     mpl11
   ];
```

found from looking at preambles in source files https://github.com/linuxmint/cjs/blob/050b695891944a4fa6c30f7aefbfdaf5b1cb0b4a/modules/_legacy.js#L8 (many)

---

licenses attributes are in `licenses.nix` btw